extension/package.json

```diff
@@ -76,7 +76,12 @@
         {
           "id": "gradleTasksView",
           "name": "Gradle Projects",
-          "when": "gradle:activated"
+          "when": "gradle:activated && !gradle:defaultView"
+        },
+        {
+          "id": "gradleDefaultProjectsView",
+          "name": "Default Gradle Projects",
```

Comment:
this name is confusing. On first glance, I feel strange about what is "Default Projects".

Reply:
suggest to use the same name, otherwise you will see flickering between "Gradle Projects" and "Default Gradle Projects" when you editing build.gradle

Reply:
Agree. Should we use something to indicate the difference between the two views (e.g. description in the project node) or just keep the same view name?

Reply:
Do you think it's necessary to let users be aware of two views? I don't think so. Implementing two views is a trick, they actually serve the same purpose and should be regarded as a single view.